packages/ui/src/components/atoms/Button/Button.stories.mdx

```diff
@@ -19,6 +19,10 @@ import { Button } from '@faststore/components'
       control: { type: 'radio' },
       description: 'Specifies the component size variant. ',
     },
+    iconPosition: {
+      options: ['left', 'right'],
+      control: { type: 'radio' },
```

Comment:
Since the `iconPosition` is optional, perhaps it would be better to treat it just as a string. What do you think?

Reply:
I didn't get it, do u mean the prop?

Reply:
I'll leave with this options, I think it's easier to understand, ok?